### ISSUE TITLE
[alchemy-language] remove protocol extensions

### ISSUE THREAD

**Issue by ruslan120101** [watson-developer-cloud/ios-sdk#119]
Transition added functionality in the 8 AlchemyLanguage methods via protocol extensions **...**

```swift
protocol Example{}
extension Example {

  func asDict() -> [String : String] { 
    ...
  }

}

struct ParamDict: Example {

  var exampleProperty = "helloWorld"

}

// (in code)
let defaultDict = ParamDict().asDict()
```

**...** to result in use of Swift **Dict**s

```swift
let exampleParamDict = ["exampleProperty" : "helloWorld"]
```

**Comment by vherrin** [comment]
I will evaluate the need for this change

**Comment by glennrfisher** [comment]
This will be addressed in #244, where we will re-implement the models to use Freddy instead.
